virttest/bootstrap.py

```diff
@@ -546,6 +546,23 @@ def create_subtests_cfg(t_type):
               subtests_cfg)
 
 
+def sync_config_lib(t_type):
+    """
+    Copy config lib from t_type/cfg/lib to backend/t_type/cfg/lib.
+
+    :param t_type: test provider, type: string
+    """
+    lib = ['cfg', 'lib']
+    dst_dir = os.path.join(data_dir.get_backend_dir(t_type), *lib)
```

Comment:
the use of `*args` should be extraordinary. Using `lib = os.path.join('cfg', 'lib')` and passing it as is would be preferred.